src/native/fs/fs_napi.cpp

```diff
@@ -99,10 +100,22 @@ struct FSWorker : public Napi::AsyncWorker
     virtual void Work() = 0;
     void Execute() override
     {
+        using std::chrono::duration_cast; 
+        using std::chrono::milliseconds;
+        using std::chrono::system_clock;
+
         DBG1("FS::FSWorker::Execute: " << _desc << DVAL(_uid) << DVAL(_gid) << DVAL(_backend));
+        if (_desc.find("FileWritev") != std::string::npos ) {
+            thresholdInMilliSec = 100;
+        }
         ThreadScope tx;
         tx.set_user(_uid, _gid);
+        std::time_t timeBefore = duration_cast<milliseconds>(system_clock::now().time_since_epoch()).count();
         Work();
+        std::time_t timeAfter = duration_cast<milliseconds>(system_clock::now().time_since_epoch()).count();
```

Comment:
This is not the way to measure elapsed time in cpp.
See https://www.geeksforgeeks.org/measure-execution-time-function-cpp/
TLDR:
```c++
auto start_time = std::chrono::high_resolution_clock::now();
Work();
auto end_time = std::chrono::high_resolution_clock::now();
double took_time = std::chrono::duration<double, std::milli>(end_time - start_time).count();
```